commit 1daec7b0b7cf0f30dc974351af5e55c3e8190370
Author: Andrew Cagney <cagney@gnu.org>
Date:   Tue Apr 12 20:02:31 2016 -0400

    Use installed antlr, don't bundle antlr.jar
    
    Patch: frysk-0.4-use-installed-antlr.jar

diff --git a/frysk-common/Makefile.gen.sh b/frysk-common/Makefile.gen.sh
index 9829184..6213a7e 100755
--- a/frysk-common/Makefile.gen.sh
+++ b/frysk-common/Makefile.gen.sh
@@ -1,6 +1,7 @@
 #!/bin/sh -eu
 # This file is part of the program FRYSK.
 #
+# Copyright 2016, Andrew Cagney
 # Copyright 2005, 2006, 2007, 2008, Red Hat Inc.
 #
 # FRYSK is free software; you can redistribute it and/or modify it
@@ -430,7 +431,6 @@ do
   echo ""
   print_header "... $jar"
   echo ${B}_JAR = @${B}_JAR@
-  print_jar_rule ${b} ${B}
 done
 
 
@@ -509,7 +509,7 @@ for suffix in .java .java-sh .mkenum .shenum .java-in ; do
 ${name}: Makefile
 	mkdir -p ${d}
 	echo "#!/bin/sh" >> ${name}.tmp
-	echo exec /usr/bin/java -cp \"\$(javadir)/*\" ${name} \"\\\$\$@\" >> ${name}.tmp
+	echo exec /usr/bin/java -cp \"\$(javadir)/*\$(INSTALLED_CLASSPATH)\" ${name} \"\\\$\$@\" >> ${name}.tmp
 	chmod a+x ${name}.tmp
 	mv ${name}.tmp ${name}
 EOF
diff --git a/frysk-common/Makefile.rules b/frysk-common/Makefile.rules
index fcc807a..84ee611 100644
--- a/frysk-common/Makefile.rules
+++ b/frysk-common/Makefile.rules
@@ -2,6 +2,7 @@
 #
 # Copyright 2005, 2006, 2007, 2008, Red Hat Inc.
 # Copyright 2007 Oracle Corporation.
+# Copyright 2016, Andrew Cagney
 #
 # FRYSK is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -139,6 +140,7 @@ ACLOCAL_AMFLAGS = -I common/m4
 
 SOURCEPATH = $(subst $(space),:,$(strip $(GEN_SOURCEPATH)))
 CLASSPATH = $(subst $(space),:,$(strip $(GEN_CLASSPATH)))
+INSTALLED_CLASSPATH = $(patsubst %,:%,$(subst $(space),:,$(filter /%.jar,$(GEN_CLASSPATH))))
 
 AM_CXXFLAGS = \
 	-I$(srcdir) \
diff --git a/frysk-core/Makefile.am b/frysk-core/Makefile.am
index 3b0a09d..fbe11ef 100644
--- a/frysk-core/Makefile.am
+++ b/frysk-core/Makefile.am
@@ -1,6 +1,7 @@
 # This file is part of the program FRYSK.
 #
 # Copyright 2005, 2007, 2008, Red Hat Inc.
+# Copyright 2016, Andrew Cagney
 #
 # FRYSK is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -43,7 +44,7 @@ include Makefile.gen
 
 GEN_CLASSPATH += ../frysk-sys/frysk-sys.jar
 GEN_CLASSPATH += ../frysk-imports/jline.jar
-GEN_CLASSPATH += ../frysk-imports/antlr.jar
+GEN_CLASSPATH += $(ANTLR_JAR)
 GEN_CLASSPATH += ../frysk-imports/junit.jar
 GEN_CLASSPATH += ../frysk-imports/getopt.jar
 
diff --git a/frysk-core/bootstrap.sh b/frysk-core/bootstrap.sh
index 03c07ef..4ed9a1c 100755
--- a/frysk-core/bootstrap.sh
+++ b/frysk-core/bootstrap.sh
@@ -2,6 +2,7 @@
 # This file is part of the program FRYSK.
 #
 # Copyright 2005, 2006, 2007, Red Hat Inc.
+# Copyright 2016, Andrew Cagney
 #
 # FRYSK is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -42,7 +43,7 @@ set -e
 
 # Generate everything (always run with --add-missing).
 
-sh -eu ./common/Makefile.gen.sh frysk prog
+sh -eu ./common/Makefile.gen.sh frysk prog ANTLR_JAR
 
 echo "Running aclocal ..."
 aclocal -I common/m4
diff --git a/frysk-core/configure.ac b/frysk-core/configure.ac
index d92c4f4..7abf364 100644
--- a/frysk-core/configure.ac
+++ b/frysk-core/configure.ac
@@ -1,6 +1,7 @@
 # This file is part of the program FRYSK.
 #
 # Copyright 2005, 2007, Red Hat Inc.
+# Copyright 2016, Andrew Cagney
 #
 # FRYSK is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -45,6 +46,7 @@ AC_INIT(frysk, [FRYSK_VERSION])
 m4_include([common/frysk-common.ac])
 
 AM_INIT_AUTOMAKE([subdir-objects foreign no-installinfo no-exeext no-dist])
+AC_FIND_FILE([antlr.jar], [/usr/share/java], ANTLR_JAR)
 
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
diff --git a/frysk-imports/bootstrap.sh b/frysk-imports/bootstrap.sh
index 83edd9b..87ddd6b 100755
--- a/frysk-imports/bootstrap.sh
+++ b/frysk-imports/bootstrap.sh
@@ -2,6 +2,7 @@
 # This file is part of the program FRYSK.
 #
 # Copyright 2005, 2006, 2007, 2008, Red Hat Inc.
+# Copyright 2016, Andrew Cagney
 #
 # FRYSK is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -42,7 +43,6 @@ set -e
 
 FILE_LIST="\
     junit/junit.jar \
-    ANTLR_JAR \
     jline/jline.jar \
     getopt/getopt.jar \
     "
@@ -50,12 +50,6 @@ FILE_LIST="\
 ./common/Makefile.gen.sh $FILE_LIST
 
 (
-    cd antlr
-    echo "Running autoconf ... for antlr"
-    autoconf -Werror -f
-)
-
-(
     cd libunwind
     mkdir -p config
     echo "Running aclocal ... for libunwind"
diff --git a/frysk-imports/configure.ac b/frysk-imports/configure.ac
index d22a250..df4ad25 100644
--- a/frysk-imports/configure.ac
+++ b/frysk-imports/configure.ac
@@ -1,6 +1,7 @@
 # This file is part of the program FRYSK.
 #
 # Copyright 2005, 2006, 2007, 2008, Red Hat Inc.
+# Copyright 2016, Andrew Cagney
 #
 # FRYSK is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -46,7 +47,6 @@ m4_include([common/frysk-common.ac])
 
 AM_INIT_AUTOMAKE([subdir-objects foreign no-installinfo no-exeext no-dist])
 
-AC_FIND_FILE([antlr.jar], [/usr/share/java /usr/share/frysk/java], ANTLR_JAR)
 AC_FIND_FILE([libaudit.h], [/usr/include/], LIBAUDIT_H)
 
 AC_CONFIG_FILES([
diff --git a/frysk-sys/Makefile.am b/frysk-sys/Makefile.am
index d121507..7465edf 100644
--- a/frysk-sys/Makefile.am
+++ b/frysk-sys/Makefile.am
@@ -1,6 +1,7 @@
 # This file is part of the program FRYSK.
 #
 # Copyright 2005, 2007, 2008, Red Hat Inc.
+# Copyright 2016, Andrew Cagney
 #
 # FRYSK is free software; you can redistribute it and/or modify it
 # under the terms of the GNU General Public License as published by
@@ -44,7 +45,6 @@ include Makefile.gen
 GEN_SOURCEPATH += ../frysk-imports/
 GEN_SOURCEPATH += $(top_srcdir)/../frysk-imports/
 GEN_CLASSPATH += ../frysk-imports/jline.jar
-GEN_CLASSPATH += ../frysk-imports/antlr.jar
 GEN_CLASSPATH += ../frysk-imports/junit.jar
 GEN_CLASSPATH += ../frysk-imports/getopt.jar
 
